ansible_collections/arista/avd/roles/eos_designs/templates/network_services/vlan-interfaces.j2

```diff
@@ -8,39 +8,19 @@ vlan_interfaces:
 {%             if svi.profile is arista.avd.defined %}
 {%                 set svi_profile = svi_profiles[svi.profile] | arista.avd.default() %}
```

Comment:
default here should be `{}`. This only comes into effect if the user points to a non-existing profile.